runtime/src/lib.rs

```diff
@@ -269,6 +269,10 @@ construct_runtime!(
         FileSystem: pallet_file_system = 41,
         ProofsDealer: pallet_proofs_dealer = 42,
         Randomness: pallet_randomness = 43,
+        BucketNfts: pallet_bucket_nfts = 44,
```

Comment:
When you merge from main, be careful because Tobi also added a pallet with this ID.